renovate.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:recommended", ":dependencyDashboard"],
```

Comment:
Do we have a plan for using the `dependencyDashboard`? This is interesting ❤️

Reply:
I can pin it